src/views/LoginView.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div
+    class="w-full min-h-screen bg-gray-50 flex flex-col sm:justify-center items-center pt-6 sm:pt-0"
+  >
+    <div class="w-full sm:max-w-md p-5 mx-auto">
+      <h2 class="mb-12 text-center text-5xl font-extrabold">Welcome.</h2>
+      <form>
+        <div class="mb-4">
+          <label class="block text-md font-light mb-2" for="username"
+            >Username</label
+          >
+          <input
+            v-model="inputUserName"
+            class="py-2 px-3 border border-gray-300 focus:border-red-300 focus:outline-none focus:ring focus:ring-red-200 focus:ring-opacity-50 rounded-md shadow-sm disabled:bg-gray-100 mt-1 block w-full"
+            type="text"
+            name="username"
+            id=""
+          />
+        </div>
+        <div class="mb-4">
+          <label class="block text-md font-light mb-2" for="password"
+            >Password</label
+          >
+          <input
+            v-model="inputPassword"
+            class="py-2 px-3 border border-gray-300 focus:border-red-300 focus:outline-none focus:ring focus:ring-red-200 focus:ring-opacity-50 rounded-md shadow-sm disabled:bg-gray-100 mt-1 block w-full"
+            type="password"
+            name="password"
+            id=""
+          />
+        </div>
+        <div class="pt-1 px-20">
+          <label
+            class="p-1 mb-5 text-sm text-red-700 bg-red-100 rounded-lg dark:bg-red-200 dark:text-red-800"
+            role="alert"
+            v-show="errorMessage"
```

Comment:
<img width="682" alt="Screenshot 2022-05-07 at 11 09 38" src="https://user-images.githubusercontent.com/71661786/167247616-fdff542d-4d20-4a32-8e76-c6a442238fc2.png">
 could we add some padding under error message?